spot-ml/ml_ops.sh

```diff
@@ -45,6 +45,14 @@ else
     RAWDATA_PATH=${PROXY_PATH}
 fi
 
+# pass the user domain designation if not empty
+
+if [ ! -z $USER_DOMAIN ] ; then
+    USER_DOMAIN_PARSER_CMD="--userdomain $USER_DOMAIN"
```

Comment:
what's with the "PARSER" in the variable name? why not USER_DOMAIN_CMD or USER_DOMAIN_CLI_ENTRY ?